experimental/emulated_vault/opt/emulated-vault/conf/cfg.json

```diff
@@ -0,0 +1,13 @@
+{
+    "data": {
+        "ping-rel-path": "ping", 
+        "db-path": "/opt/emulated-vault/db/",
+        "adaptor-type": "fs",
+        "listen-ip": "0.0.0.0",
+        "listen-port": 8088,
+        "use-ssl": true,
+        "ssl-key-path": "path/to/cert/key.pem",
+        "ssl-cert-path": "path/to/cert/cert.pem"
+    }, 
+    "format-version": 100
+}
```

Comment:
Instead of using JSON, you could maybe make this a bit easier on yourself by using [the Python standard library's configuraiton file format](https://docs.python.org/3/library/configparser.html)